### PR TITLE
Method to generate fake data and seed into database created 

### PR DESCRIPTION
**Pull request by ayahusseini**
Script functionality:
- Connects to the database 
- generates fake subscribers, topics, mappings, to real headlines
- fake polarity score, sampled randomly but biased towards the negative (for fox news)

possible future improvements:
- functions have parameters specifying number of fake data rows to generate already, so could include this as a CLI parameter. 
- Need to double check queries against the creating tables schema
- Assumes sources have already been seeded. Need to check this.

### CODE REVIEW

**Comment by ayahusseini** [comment]
ERD Changing: need to update before approval

File: database/generate_dummy_data.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+'''Generates dummy data'''
+from os import environ as ENV
+import random
+from faker import Faker
+
+from dotenv import load_dotenv
+import psycopg2
+from psycopg2 import extras
+
+TOPICS = [
```
Comment (megansam5) [inline database/generate_dummy_data.py:10]: Well done changing the topics

**Review by indiah444 — APPROVED** [review]
This all looks brilliant, this should generate some good fake data for testing later on, thank you!

**Review by megansam5 — APPROVED** [review]
Nice!

**Pull request merged**